packages/@okta/vuepress-site/docs/reference/api/zones/index.md

```diff
@@ -115,6 +115,8 @@ Each Location object specifies an [ISO-3166-1](https://en.wikipedia.org/wiki/ISO
 | country        | Format of the value: length 2 ISO-3166-1 country code                | String      | Yes       |
 | region       | Format of the value: `countryCode`-`regionCode`, or `null` if empty  | String      | No       |
 
+> **Note**: Continent codes are not intended to be used as region definitions. The EU and AP codes are only used if a specific country code has not been set. To include all of the countries in Europe or in Asia/Pacific, each country needs to be specified individually.  Used alone, Europe and Asia/Pacific are treated as generic codes for undesignated regions rather than inclusive of the countries they contain.
```

Comment:
Can you provide an example of a continent code in lieu of country code? It is unclear what the format should be?
"country": null,
"region": "-AP" ?
"region": "AP" ?
Also, is `regionCode` == continent code?

Reply:
I'm reading the Jira ticket and for the API docs, I think we may need to clarify what type of data is allowed in the Location object. Suggestion to update the Location object property table:

| Field Name  | Description                                                | Data Type   | Required |
| :---------- | :--------------------------------------------------------- | :---------- | :------- |
| country        | The country included in the Zone perimeter. Specify the [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code.               | String      | Yes       |
| region       | The country subdivision included in the Zone perimeter. Specify the [ISO-3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) region code, in the format: `countryCode`-`regionCode`. This field is optional: if you specify the subdivision region code, then only this region is considered in the Zone perimeter; if you specify `null` in this field, then the entire country is considered in the Zone perimeter.   | String      | No       |

> **Note:** You cannot specify a continent code in the `country` or `region` fields, for example `EU` for Europe. If you want to include all of the countries in a continent, you need to specify a location for each country. If you specify an unknown country or region code, then the Zone perimeter does not include any location.